pkg/scalers/azure_monitor.go

```diff
@@ -0,0 +1,200 @@
+package scalers
```

Comment:
May be worth specifically calling out the Azure Kubernetes Metric Adapter codebase in this file - I know you mentioned it in the PR but for future readers putting the credit/reference in the codebase itself may be helpful

Reply:
Good call.